extensions/amp-analytics/0.1/test/test-session-manager.js

```diff
@@ -129,6 +129,34 @@ describes.realWin('Session Manager', {amp: true}, (env) => {
       );
     });
 
+    it('should not change creationTimestamp on update', async () => {
```

Comment:
Should the test verify that the `lastAccessTimestamp` is updated?

Reply:
The test `should update an existing session in memory and storage`, tests for this (it's the same test, except that in this one, I explicitly call out that we must not update creationTimestamp).